### PR TITLE
[IMP][8.0] website_snippet_contact_form: Minor improvements to select file upload options

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Yajo

### CODE REVIEW

File: website_snippet_contact_form/i18n/es.po [hunk 1]
```diff
@@ -103,3 +108,6 @@ msgstr "Su nombre"
 #: view:website:website.snippets
 msgid "Your Question"
 msgstr "Su pregunta"
+
+#~ msgid "Allow Attachment"
```
Comment (pedrobaeza) [inline website_snippet_contact_form/i18n/es.po]: Why this commented?

**Comment by rafaelbn** [comment]
Tested functionally in runbot :+1:

**Comment by sergio-teruel** [comment]
@pedrobaeza changes done.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by carlosdauden** [comment]
:+1:

**Pull request merged**